src/types.d.ts

```diff
@@ -0,0 +1,3 @@
+declare module '*.png' {
```

Comment:
Tuto změnu můžeme klidně zanechat, ale v případě ikony jsem přemýšlel, zda nemůžeme využít SVG formát jako React komponentu. Např. při použití [tohoto convertoru](https://react-svgr.com/playground/). Co myslíš?

Reply:
Dobrý nápad! 👍  Mrknu na to :)

Reply:
Hotovo.